components/AppBar/src/MDCAppBarNavigationController.h

```diff
@@ -131,6 +131,14 @@ __attribute__((objc_subclassing_restricted)) @interface MDCAppBarNavigationContr
 - (nullable MDCAppBarViewController *)appBarViewControllerForViewController:
     (nonnull UIViewController *)viewController;
 
+/**
+ A block that is assigned to each assigned to each @c MDCAppBarViewController's
```

Comment:
i think there is a bit of a duplication in wording here

Reply:
yeah

Reply:
Done.